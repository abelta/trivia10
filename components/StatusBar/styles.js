import { StatusBar, StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  statusBar: {
    height: StatusBar.currentHeight,
  },
});

export default styles;
