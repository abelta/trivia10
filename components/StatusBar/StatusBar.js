import React from 'react';
import { StatusBar, View } from 'react-native';
import styles from './styles';

const StatusBarCustom = () => (
  <View style={styles.statusBar}>
    <StatusBar
      backgroundColor="black"
      barStyle="light-content"
      translucent={false}
      hidden={false}
    />
  </View>
);

export default StatusBarCustom;
