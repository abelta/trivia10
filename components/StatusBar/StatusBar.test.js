import React from 'react';
import renderer from 'react-test-renderer';
import StatusBar from '.';

describe('StatusBar', () => {
  it('renders properly', () => {
    const tree = renderer.create(<StatusBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
