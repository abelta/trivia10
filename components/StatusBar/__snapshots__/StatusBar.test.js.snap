// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar renders properly 1`] = `
<View
  style={
    Object {
      "height": undefined,
    }
  }
/>
`;
